package-lock.json

```diff
@@ -125,7 +125,7 @@
         "ws": "^7.2.1"
       },
       "engines": {
-        "node": "^8.6.0 || 10 || 12 || 14 || 16 || 17 || 18 || 19"
+        "node": ">=8.6.0"
```

Comment:
This part of the update should have actually come from the recent v20-related commit where I updated package.json but didn't `npm install` to update the lock file.